Titan: CMake <3.4.0 & -ccbin in FindCUDA

The build logs are [here](https://gist.github.com/anshumang/e3e248e8a6027676bbc7) and [here](https://gist.github.com/anshumang/194984b96ee0d789fcb9) on dev.
## 

[Jump to Solution/Work-Around](https://github.com/ComputationalRadiationPhysics/picongpu/issues/1060#issuecomment-141029045)

@ax3l Still getting an error even with an interactive session....:grimacing:

anshuman@titan-login8:~/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009> make VERBOSE=1 -j32
/autofs/nccs-svm1_sw/xk6/cmake3/3.1.0/sles11.1_gnu4.3.4/bin/cmake -H/ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/picongpu -B/ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009 --check-build-system CMakeFiles/Makefile.cmake 0
/autofs/nccs-svm1_sw/xk6/cmake3/3.1.0/sles11.1_gnu4.3.4/bin/cmake -E cmake_progress_start /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/CMakeFiles /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/CMakeFiles/progress.marks
make -f CMakeFiles/Makefile2 all
make[1]: Entering directory `/autofs/nccs-svm1_home1/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009'
make -f build_picongpu/CMakeFiles/picongpu.dir/build.make build_picongpu/CMakeFiles/picongpu.dir/depend
make[2]: Entering directory `/autofs/nccs-svm1_home1/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009'
/autofs/nccs-svm1_sw/xk6/cmake3/3.1.0/sles11.1_gnu4.3.4/bin/cmake -E cmake_progress_report /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/CMakeFiles 1
[ 50%] Building NVCC (Device) object build_picongpu/CMakeFiles/picongpu.dir/picongpu_generated_main.cu.o
cd /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/build_picongpu/CMakeFiles/picongpu.dir && /autofs/nccs-svm1_sw/xk6/cmake3/3.1.0/sles11.1_gnu4.3.4/bin/cmake -E make_directory /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/build_picongpu/CMakeFiles/picongpu.dir//.
cd /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/build_picongpu/CMakeFiles/picongpu.dir && /autofs/nccs-svm1_sw/xk6/cmake3/3.1.0/sles11.1_gnu4.3.4/bin/cmake -D verbose:BOOL=1 -D build_configuration:STRING=Debug -D generated_file:STRING=/ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/build_picongpu/CMakeFiles/picongpu.dir//./picongpu_generated_main.cu.o -D generated_cubin_file:STRING=/ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/build_picongpu/CMakeFiles/picongpu.dir//./picongpu_generated_main.cu.o.cubin.txt -P /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/build_picongpu/CMakeFiles/picongpu.dir//picongpu_generated_main.cu.o.cmake
-- Removing /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/build_picongpu/CMakeFiles/picongpu.dir//./picongpu_generated_main.cu.o
/autofs/nccs-svm1_sw/xk6/cmake3/3.1.0/sles11.1_gnu4.3.4/bin/cmake -E remove /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/build_picongpu/CMakeFiles/picongpu.dir//./picongpu_generated_main.cu.o
-- Generating dependency file: /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/build_picongpu/CMakeFiles/picongpu.dir//picongpu_generated_main.cu.o.NVCC-depend
/opt/nvidia/cudatoolkit6.5/6.5.14-1.0502.9613.6.1/bin/nvcc -M -D__CUDACC__ /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/picongpu/src/picongpu/main.cu -o /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/build_picongpu/CMakeFiles/picongpu.dir//picongpu_generated_main.cu.o.NVCC-depend -ccbin /opt/cray/craype/2.4.0/bin/driver -m64 -DPMACC_VERBOSE_LVL=0 -DPIC_VERBOSE_LVL=1 -Xcompiler ,\"-fopenmp\",\"-Wall\",\"-Wno-unknown-pragmas\",\"-Wextra\",\"-Wno-unused-parameter\",\"-Wno-unused-local-typedefs\",\"-g\" -arch=sm_20 --use_fast_math --ftz=false -g -Xcompiler=-g,-pthread -DNVCC -I/opt/nvidia/cudatoolkit6.5/6.5.14-1.0502.9613.6.1/include -I/ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/picongpu/examples/KelvinHelmholtz/include -I/sw/xk6/boost/1.57.0/cle5.2_gnu4.8.2/include -I/ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/picongpu/thirdParty/mallocMC/src/include -I/ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/picongpu/src/picongpu/../libPMacc/include -I/ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/picongpu/src/picongpu/include -I/opt/nvidia/cudatoolkit6.5/6.5.14-1.0502.9613.6.1/include
CMake Error at picongpu_generated_main.cu.o.cmake:206 (message):
  Error generating
  /ccs/home/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009/build_picongpu/CMakeFiles/picongpu.dir//./picongpu_generated_main.cu.o


make[2]: *** [build_picongpu/CMakeFiles/picongpu.dir/picongpu_generated_main.cu.o] Error 1
make[2]: Leaving directory `/autofs/nccs-svm1_home1/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009'
make[1]: *** [build_picongpu/CMakeFiles/picongpu.dir/all] Error 2
make[1]: Leaving directory `/autofs/nccs-svm1_home1/anshuman/ComputationalRadiationPhysics/picongpu-repos/08-25-2015/builds/0009'
make: *** [all] Error 2

--- a/src/picongpu/submit/titan-ornl/picongpu.profile.example
+++ b/src/picongpu/submit/titan-ornl/picongpu.profile.example
@@ -3,14 +3,15 @@ export TBG_mailAdress="mail@example.com"
 
 # basic environment #################################################
 source /opt/modules/3.2.6.7/init/bash
+module load craype-accel-nvidia35
 module swap PrgEnv-pgi PrgEnv-gnu
 # module swap gcc gcc/4.8.2 # default
 
 # Compile for CLE nodes
 #   (CMake likes to unwrap the Cray wrappers)
-export CC="`which cc` -target=compute_node"
-export CXX="`which CC` -target=compute_node"
-export FC="`which ftn` -target=compute_node"
+export CC=`which cc`
+export CXX=`which CC`
+export FC=`which ftn`
 #export LD="/sw/xk6/altd/bin/ld"
 
 # symbol bug work around (should not be required)

